src/one_dragon/utils/cv2_utils.py

```
@@ -1234,3 +1235,54 @@ def is_colorful(img: MatLike, saturation_threshold: int = 30, color_ratio_thresh
     is_colorful = mean_saturation > saturation_threshold and color_ratio > color_ratio_threshold
 
     return is_colorful
+
+
+def is_in_gray_mask(img: MatLike, rect: Rect | None = None, threshold: int = 55, rgb_diff_threshold: int = 20,
+                    percent_rate: float = 0.9) -> bool:
+    """
+    判断图像是否接近灰色遮罩下的区域, 可用于判断游戏下方的对话框遮罩
+    性能比 is_colorful 稍慢, 但是几乎能准确识别所有对话框
+
+    判断依据：
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)